### PR TITLE
[Doc EN]: `guide/plugins.md`

### PR DESCRIPTION
**Pull request by Haeresis**
In our French translation/review we have changed some consistent things, so we suggest you do the same in the original documentation.

More details in comments.

Do not hesitate to set back the original behavior for some changes if you think they do not make sense.

### CODE REVIEW

**Comment by Haeresis** [comment]
Who is in charge of documentation PR. There is something I can do to help?

**Comment by Haeresis** [comment]
HI @Atinux !

Who is in charge of documentation PR. There is something I can do to help?

File: en/guide/plugins.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Plugins
-description: Nuxt.js allows you to define JavaScript plugins to be run before instantiating the root vue.js application. This is especially helpful when using your own libraries or external modules.
+description: Nuxt.js allows you to define JavaScript plugins to be run before instantiating the root Vue.js Application. This is especially helpful when using your own libraries or external modules.
```
Comment (Haeresis) [inline en/guide/plugins.md]: To be more consistent with « Universals Vue.js Applications »

File: en/guide/plugins.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ Then, I can import `axios` anywhere without having to worry about making the bun
 If we want to use [vue-notifications](https://github.com/se-panfilov/vue-notifications) to display notification in our application, we need to setup the plugin before launching the app.
 
 File `plugins/vue-notifications.js`:
+
```
Comment (Haeresis) [inline en/guide/plugins.md]: By convention, Markdown use new line between each new paragraph/code block

File: en/guide/plugins.md [hunk 3]
```diff
@@ -61,6 +62,7 @@ Vue.use(VueNotifications)
 ```
 
 Then, we add the file inside the `plugins` key of `nuxt.config.js`:
+
```
Comment (Haeresis) [inline en/guide/plugins.md]: By convention, Markdown use new line between each new paragraph/code block

File: en/guide/plugins.md [hunk 4]
```diff
@@ -72,6 +74,7 @@ To learn more about the `plugins` configuration key, check out the [plugins api]
 Actually, `vue-notifications` will be included in the app bundle, but because it's a library, we want to include it in the vendor bundle for better caching.
 
 We can update our `nuxt.config.js` to add `vue-notifications` in the vendor bundle:
+
```
Comment (Haeresis) [inline en/guide/plugins.md]: By convention, Markdown use new line between each new paragraph/code block

File: en/guide/plugins.md [hunk 5]
```diff
@@ -86,6 +89,7 @@ module.exports = {
 Some plugins need to be injected in the App root to be used, like [vue-18n](https://github.com/kazupon/vue-i18n). Nuxt.js gives you the possibility to export a function in your plugin to receives the root component but also the context.
 
 `plugins/i18n.js`:
+
```
Comment (Haeresis) [inline en/guide/plugins.md]: By convention, Markdown use new line between each new paragraph/code block

File: en/guide/plugins.md [hunk 6]
```diff
@@ -102,6 +106,7 @@ export default ({ app, store }) => {
 ```
 
 `nuxt.config.js`:
+
```
Comment (Haeresis) [inline en/guide/plugins.md]: By convention, Markdown use new line between each new paragraph/code block

File: en/guide/plugins.md [hunk 7]
```diff
@@ -120,6 +125,7 @@ Some plugins might work **only for the browser**, you can use the `ssr: false` o
 Example:
 
 `nuxt.config.js`:
+
```
Comment (Haeresis) [inline en/guide/plugins.md]: By convention, Markdown use new line between each new paragraph/code block

File: en/guide/plugins.md [hunk 8]
```diff
@@ -129,6 +135,7 @@ module.exports = {
 ```
 
 `plugins/vue-notifications.js`:
+
```
Comment (Haeresis) [inline en/guide/plugins.md]: By convention, Markdown use new line between each new paragraph/code block

**Comment by Atinux** [comment]
I am in charge of PR for english, thank you @Haeresis it's merged!

**Pull request merged**